android/src/com/mapswithme/maps/widget/placepage/EditDescriptionFragment.java

```diff
@@ -1,125 +0,0 @@
-package com.mapswithme.maps.widget.placepage;
-
-import android.os.Bundle;
-import android.text.Html;
-import android.view.LayoutInflater;
-import android.view.View;
-import android.view.ViewGroup;
-import android.view.WindowManager;
-import android.widget.EditText;
-import android.widget.TextView;
-
-import androidx.annotation.NonNull;
-import androidx.annotation.Nullable;
-import androidx.appcompat.widget.Toolbar;
-
-import com.mapswithme.maps.R;
-import com.mapswithme.maps.base.BaseMwmDialogFragment;
-import com.mapswithme.maps.bookmarks.data.Bookmark;
-import com.mapswithme.util.StringUtils;
-import com.mapswithme.util.UiUtils;
-
-import java.lang.ref.WeakReference;
-
-public class EditDescriptionFragment extends BaseMwmDialogFragment
-{
-  public static final String EXTRA_BOOKMARK = "bookmark";
-
-  private EditText mEtDescription;
-  private Bookmark mBookmark;
-
-  public interface OnDescriptionSavedListener
-  {
-    void onSaved(Bookmark bookmark);
-  }
-
-  private WeakReference<OnDescriptionSavedListener> mListener;
-
-  public EditDescriptionFragment() {}
-
-  @Override
-  protected int getCustomTheme()
-  {
-    return getFullscreenTheme();
-  }
-
-  @Nullable
-  @Override
-  public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState)
-  {
-    return inflater.inflate(R.layout.fragment_edit_description, container, false);
-  }
-
-  @Override
-  public void onViewCreated(@NonNull View view, Bundle savedInstanceState)
-  {
-    mBookmark = getArguments().getParcelable(EXTRA_BOOKMARK);
-    String description = null;
-    if (mBookmark != null)
-      description = mBookmark.getBookmarkDescription();
-
-    if (description != null && StringUtils.nativeIsHtml(description))
-    {
-      final String descriptionNoSimpleTags = StringUtils.removeEditTextHtmlTags(description);
-      if (!StringUtils.nativeIsHtml(descriptionNoSimpleTags))
-        description = Html.fromHtml(description).toString();
```

Comment:
Wait, is the bookmark's description now editable?

Reply:
Yes I really don't know what this class was used for